ci/requirements/docs.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
     - python=3.12
     - gmt=6.5.0
     - ghostscript=10.02.1
-    - numpy>=1.22
```

Comment:
Removing the numpy version information from this file, so that we have one fewer file to update next time.

Reply:
Ok, using `>=1.22` might mean the latest numpy version is pulled in anyways.